e2e/testsuite/testconfig.go

```diff
@@ -723,3 +731,17 @@ func modifyClientGenesisAppState(ibcAppState []byte) ([]byte, error) {
 
 	return ibcGenBz, nil
 }
+
+// modifyChannelGenesisAppState takes the existing ibc app state and marshals it to a ibc GenesisState.
```

Comment:
want to add shenanigans you do in function to docstring? Its same as other modify one, might be nice to drop a line here